packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.jsx

```diff
@@ -20,6 +20,17 @@ const ProofOfAddressContainer = ({
     has_restricted_mt5_account,
     refreshNotifications,
     app_routing_history,
+    account_settings,
+    addNotificationByKey,
+    is_eu,
+    is_verification_modal_visible,
+    fetchResidenceList,
+    fetchStatesList,
+    index,
+    onSubmit,
+    removeNotificationByKey,
+    removeNotificationMessage,
+    states_list,
```

Comment:
These additional props are just passed to ProofOfAddressForm. rigt?
Can we just pass these with spread operator?

```
const ProofOfAddressContainer = ({
    is_mx_mlt,
    is_switching,
    has_restricted_mt5_account,
    refreshNotifications,
    app_routing_history,
    ...props
})

```

And in
 ```
<ProofOfAddressForm
                is_resubmit
                onSubmitting={() => onSubmitValues({ needs_poi })}
                {...props}
                />
```